client/src/components/ButtonsBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FunctionComponent } from "react";
+import { Button, Div, Icon } from "atomize";
+import MicIcon from "@material-ui/icons/Mic";
+import MicOffIcon from "@material-ui/icons/MicOff";
+
+type ButtonsBarProps = {
+  settingsCallback: () => void,
+  statusCallback: () => void,
+  muteCallback: () => void,
+  leaveCallback: () => void,
+  userInfoRef: any
+}
+
+export const ButtonsBar: FunctionComponent<ButtonsBarProps> = (props) => {
+  const defaultButtonStyle = {
+    p: "1rem",
+    m: "0.3rem",
+    bg: "none",
+    h: "2.5rem",
+    w: "2.5rem",
+    hoverBg: "#ffffff29",
+    rounded: "circle",
+  };
+  
+  function SettingsButton() {
```

Comment:
I still don't know the difference between the classic `function` declaration vs arrow function.
If there is no need to use `function`, I think it would be nice to unify React Components to arrow function.

---

I'll just put up a suggestion for the ease in case of changing it to arrow function.

```suggestion
  const SettingsButton = () => {
```

Reply:
It's been a while since I read it so I might be off on something, but check out https://github.com/getify/You-Dont-Know-JS for a better explanation of all this.

Reply:
I just quickly went through this https://dmitripavlutin.com/differences-between-arrow-and-regular-functions/
and I don't think there'd be any issues